src/issues/issueFeatureRegistrar.ts

```diff
@@ -465,6 +465,10 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
 
 	private async makeNewIssueFile(title?: string, body?: string, assignees?: string[] | undefined) {
 		const bodyPath = vscode.Uri.parse(`${NEW_ISSUE_SCHEME}:/NewIssue.md`);
+		if (vscode.window.visibleTextEditors.filter(editor => editor.document.uri.scheme === NEW_ISSUE_SCHEME).length > 0) {
+			return;
+		}
+		await vscode.workspace.fs.delete(bodyPath);

```

Comment:
not sure I understand this line, why is this needed?

Reply:
It seemed like the easiest way to reset the file and get rid of the decorations. If the file is still open in a tab that isn't currently visible, the decorations will still be there and can end up in the wrong place.